common/test/acceptance/tests/lms/test_lms.py

```diff
@@ -320,6 +359,15 @@ def test_enrollment_upgrade(self):
         enrollment_mode = self.dashboard_page.get_enrollment_mode(self.course_info["display_name"])
         self.assertEqual(enrollment_mode, 'verified')
 
+        # Expect no enrollment activated event
+        assert_event_emitted_num_times(
+            self.event_collection,
+            "edx.course.enrollment.activated",
```

Comment:
In this test, this event is emitted immediately after the auto auth page is visited. That being the case, I'd move this up to  immediately after `student_id = AutoAuthPage(self.browser, course_id=self.course_id).visit().get_user_id()`.